mesonbuild/templates/cudatemplates.py

```diff
@@ -36,9 +36,13 @@
 dependencies = [{dependencies}
 ]
 
+sources = [{source_files}
+
+]
+
 exe = executable(
   '{exe_name}',
-  '{source_name}',
+  [sources],
```

Comment:
Why the square brackets? The list is flattened, but I don't see the reason for the nested list.